main.py

```diff
@@ -92,6 +108,15 @@ def main():
     with WATCH_HISTORY_FILE.open(encoding="utf8") as f:
         data = json.load(f)
 
+    if REMOVE_SHORTS is True:
+        popped = 0
+        for index in reversed(range(len(data))):
+            if data[index]["title"].lower().find("short") > 0:
+                data.pop(index)
+                print(f"popped {data[index]['title']}")
+                popped += 1
+        print(f"total popped {popped}")
+
```

Comment:
Suggestions:
 - Better wording for the prints
 - `if REMOVE_SHORTS is True` -> `if REMOVE_SHORTS`
 - I'm not sure why this needs to be handled here and not in `filter_video_events` ? This looks more readable to me:
```
if '#shorts' in event['title']:
    continue
```

Reply:
Removed entirely